### PR TITLE
Adding mailgun tests. Fixes #181

### LINKED ISSUE #181: managed_vms/mailgun needs tests

### PR DESCRIPTION
**Pull request by jonparrott**
/cc @waprin

### CODE REVIEW

File: managed_vms/mailgun/main_test.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All rights reserved.
```
Comment (jerjou) [inline managed_vms/mailgun/main_test.py]: 2016?

Comment (jonparrott) [inline managed_vms/mailgun/main_test.py]: Copied a test from 2015. I think I need to leave it alone.

File: managed_vms/mailgun/main_test.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+@responses.activate
+def test_send_error(app):
+    responses.add(
+        responses.POST,
+        re.compile(r'.*'),
+        body='Test error',
+        status=500)
+
+    with pytest.raises(Exception):
```
Comment (jerjou) [inline managed_vms/mailgun/main_test.py]: Should probably have a more specific Exception

File: managed_vms/mailgun/main_test.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+def test_index(app):
+    r = app.get('/')
+    assert r.status_code == 200
+
+
+@responses.activate
+def test_send_error(app):
+    responses.add(
+        responses.POST,
+        re.compile(r'.*'),
```
Comment (jerjou) [inline managed_vms/mailgun/main_test.py]: Can you specify at least the domain or path for the expected endpoint?

File: managed_vms/mailgun/main_test.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        body='Test error',
+        status=500)
+
+    with pytest.raises(Exception):
+        app.post('/send/email', data={
+            'recipient': 'user@example.com',
+            'submit': 'Send simple email'})
```
Comment (jerjou) [inline managed_vms/mailgun/main_test.py]: Can you assert len(responses.call)?
(here and below)

File: managed_vms/mailgun/main_test.py (hunk 3 above)
Comment (jonparrott) [inline managed_vms/mailgun/main_test.py]: done

File: managed_vms/mailgun/main_test.py (hunk 2 above)
Comment (jonparrott) [inline managed_vms/mailgun/main_test.py]: done

File: managed_vms/mailgun/main_test.py (hunk 4 above)
Comment (jonparrott) [inline managed_vms/mailgun/main_test.py]: done

**Comment by jerjou** [comment]
LGTM

**Pull request merged**